internal/providers/terraform/azure/storage_account_customer_managed_key.go

```diff
@@ -0,0 +1,20 @@
+package azure
+
+import (
+	"github.com/infracost/infracost/internal/schema"
+)
+
+func getStorageAccountCustomerManagedKeyRegistryItem() *schema.RegistryItem {
```

Comment:
I added this free resource so we can look it up from the storage account. As per [here](https://azure.microsoft.com/en-us/pricing/details/storage/tables/) there's different prices if there's a managed key so we need to be able to connect `azurerm_storage_table -> `azurerm_storage_account` -> `azurerm_storage_account_customer_managed_key`.